src/PostgREST/Config/Database.hs

```diff
@@ -64,6 +64,7 @@ dbSettingsNames =
   ,"openapi_server_proxy_uri"
   ,"raw_media_types"
   ,"server_trace_header"
+  ,"server_timing_enabled"
```

Comment:
Added this for the in-db config to work.

I do realize that it's easy to miss it since it's not part of the Config types. Will open an issue to fix this.